tests/time_tests/test_runner/conftest.py

```diff
@@ -96,3 +124,45 @@ def get_dict_values(d):
     values = list(get_dict_values(val))
 
     return "-".join(["_".join([key, val]) for key, val in zip(keys, values)])
+
+
+@pytest.mark.hookwrapper
+def pytest_runtest_makereport(item, call):
+    """Pytest hook for report preparation.
+
+    Submit tests' data to a database.
+    """
+
+    KEY_FIELDS = ('timetest', 'model', 'device', 'niter', 'run_id')
+
+    run_id = item.config.getoption("db_submit")
+    db_url = item.config.getoption("db_url")
+    db_collection = item.config.getoption("db_collection")
+    if not (run_id and db_url and db_collection):
+        yield
+        return
+
+    data = item.funcargs.copy()
+    data["timetest"] = data.pop("executable").stem
+    data.update({key: val for key, val in data["instance"].items()})
+    data.pop("instance")
+    data.pop("request")
```

Comment:
Should we explicitly add required key fields? Non-blocking though.

Reply:
Added `FIELDS_FOR_SUBMIT`. Only these fields will be presented in database records